### DISCUSSION TITLE
Does the module support invoices from POS.

### DISCUSSION THREAD

**Discussion by BOTSolutions8686** [lavaloon-eg/ksa_compliance#187] [Q&A]
Hi. I have seen a previous discussion from july where someone asked if the module supports POS invoices. And reply was that its in the roadmap. 

I would like to know if the feature has been implemented by now.

**Comment by mhaggag** [comment]
Hello,

Yes, it's available since 0.32.2 ([reference issue](https://github.com/lavaloon-eg/ksa_compliance/issues/69#issuecomment-2340100049) and [PR](https://github.com/lavaloon-eg/ksa_compliance/pull/111))
